orcid-core/src/main/java/org/orcid/core/manager/impl/OrcidProfileManagerImpl.java

```diff
@@ -1986,6 +1992,16 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
                     orcidIndexManager.persistProfileInformationForIndexingIfNecessary(orcidProfile);
                     profileDao.updateIndexingStatus(orcid, IndexingStatus.DONE);
                 }
+                
+                //TODO: XXX send the message through the message queue
```

Comment:
Messages are already being sent to the queue by profile last modified aspect, so this code can be removed.